src/Store.ts

```diff
@@ -108,7 +110,7 @@ abstract class Store extends Dispatcher {
      * @returns {Function} call the function and release handler
      * @public
      */
-    onChange(cb: Function): () => void {
+    onChange(cb: (hangingStores: Array<Store>) => void): () => void {
```

Comment:
Yes. It is reasonable.
If you make it `StoreLike`, you should change `emitChange` implemenration.

```js
this.emit(STATE_CHANGE_EVENT,  [this]);
```